### ISSUE TITLE
nlcd is required for example 3

### ISSUE THREAD

**Issue by sjsrey** [spatialucr/geosnap#138]
03_harmonizing_community_boundaries

should this be documented somewhere as it isn't clear what to do (the error is going to be cryptic for an end user):

```
~/geosnap/harmonize/harmonize.py in harmonize(raw_community, target_year, weights_method, extensive_variables, intensive_variables, allocate_total, raster, codes, force_crs_match, index, time_col)
    145                 raster_path=raster,
    146                 codes=codes,
--> 147                 force_crs_match=force_crs_match,
    148             )
    149 

~/tobler/tobler/area_weighted.py in area_tables_raster(source_df, target_df, raster_path, codes, force_crs_match)
    443     # The 'append_profile_in_gdf' function is present in nlcd.py script
    444     res_union = fast_append_profile_in_gdf(
--> 445         res_union_pre, raster_path, force_crs_match=force_crs_match
    446     )
    447 

~/tobler/tobler/vectorized_raster_interpolation.py in fast_append_profile_in_gdf(geodataframe, raster_path, force_crs_match)
    169 
    170     _check_presence_of_crs(geodataframe)
--> 171     raster_path = fetch_quilt_path(raster_path)
    172     if force_crs_match:
    173         raster = rasterio.open(raster_path)

~/tobler/tobler/data.py in fetch_quilt_path(path)
     39         except ImportError:
     40             raise(
---> 41                 "Unable to locate local raster data. If you would like to use "
     42                 "raster data from the National Land Cover Database, you can "
     43                 "store it locally using the `data.store_rasters()` function"

TypeError: exceptions must derive from BaseException
```

**Comment by knaaptime** [comment]
yeah, thats a mistake anyway, as it needs to raise a real exception so the text actually prints. maybe `IOError`?

**Comment by knaaptime** [comment]
resolved bt #158
